snap/snapcraft.yaml

```diff
@@ -9,6 +9,9 @@ apps:
   heroku:
     command: bin/run
 
+architectures:
+  - build-on: [amd64, armhf]
+
```

Comment:
The default value for `run-on` is the value of `build-on`, which is why it's been omitted.